solution/ui/regulations/css/scss/partials/_header.scss

```diff
@@ -209,15 +211,19 @@ header {
 
                     &.links__list--wide {
                         display: flex;
+                        flex-direction: row;
+                        gap: 1.5rem;
                         align-items: center;
+                        margin-block: 0px;
 
-                        @include custom-max(($eds-width-sm - 1) / 1px) {
+                        @include custom-max(
+                            calc(calc($eds-width-sm - 1px) / 1px)
+                        ) {
```

Comment:
these custom min/max mixins have been updated to address SCSS warnings that were constantly firing during the local SCSS to CSS compilation build step.  So in that sense this change is good.

However, the double `calc` usage makes these mixins gross to look at.  These will probably be addressed in a future story, either to greatly simplify the mixins or to remove them entirely in exchange for a more vanilla CSS solution.